src/main/java/org/ibp/api/rest/feedback/FeedbackResource.java

```diff
@@ -32,4 +32,11 @@ public ResponseEntity<Void> dontShowAgain(@PathVariable final FeedbackFeature fe
 		return new ResponseEntity<>(HttpStatus.OK);
 	}
 
+	@ApiIgnore
+	@RequestMapping(value = "/{feature}/collectorId", method = RequestMethod.GET)
```

Comment:
@cuenyad GET feedback/{feature} shoud return the feedback entity including any other relevant information